### ISSUE TITLE
Shows all pages have changed when coming back from standby mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [max99x/pagemon-chrome-ext#8]
```
What steps will reproduce the problem?
1. Chrome is open and running
2. Place laptop in standby mode
3.

What is the expected output? What do you see instead?

The pagemonitor extension upon awakening the computer from sleep/standby 
shows that all monitored pages have changed when in fact they have not.

What version of the product are you using? On what operating system?
v2.0 on Win XP

Please provide any additional information below.


```

Original issue reported on code.google.com by `Jwm...@gmail.com` on 11 Feb 2010 at 10:48

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I could not reproduce this error. Does this happen once or each time you return 
from 
standby? Could you please attach a sample localStorage file where this is 
happening? 
You can get the file by searching for "chrome-
extension_pemhgklkefakciniebenbfclihhmmfcd_0.localstorage" in your Chrome 
folder.

Note that this bug may be irrelevant if the system described in Issue 4 
(http://code.google.com/p/pagemon-chrome-ext/issues/detail?id=4) is 
implemented.
```

Original comment by `max99x` on 12 Feb 2010 at 8:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `max99x` on 12 Feb 2010 at 8:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `max99x` on 12 Feb 2010 at 6:35
* Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Yes, it shows at least one page change alert each time I go into standby and 
then come 
back out. I check the pages and there is no change.

Also some weird stuff has started happening with the new 2.x versions. See the 
attached 
screen shot for an example. See anything funny? Like the time settings?
```

Original comment by `Jwm...@gmail.com` on 12 Feb 2010 at 6:37

Attachments:
* [chrome-extension_pemhgklkefakciniebenbfclihhmmfcd_0.localstorage](https://storage.googleapis.com/google-code-attachments/pagemon-chrome-ext/issue-8/comment-5/chrome-extension_pemhgklkefakciniebenbfclihhmmfcd_0.localstorage)
* [Problem.jpg](https://storage.googleapis.com/google-code-attachments/pagemon-chrome-ext/issue-8/comment-5/Problem.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Fixed; will be released later today in v1.0.2. For the record, apparently, 
jQuery 
(v1.4.1) considers a failure due to network inaccessibility a success and calls 
the 
success handler, breaking the "is network up?" check.

Regarding the last check in your screenshot, if the page is marked as updated, 
it will 
not be checked again until unmarked. If the page in the screenshot has been 
marked as 
visited for more than 1 minute 15 seconds (or wasn't updated in the first 
place), then 
it's a bug; please open a new issue for it.
```

Original comment by `max99x` on 12 Feb 2010 at 7:02
* Changed state: **Fixed**
